src/DssSpell.t.sol

```diff
@@ -139,6 +192,27 @@ contract DssSpellTest is DSTest, DSMath {
     ClipperMomAbstract   clipMom = ClipperMomAbstract( addr.addr("CLIPPER_MOM"));
     DssAutoLineAbstract autoLine = DssAutoLineAbstract(addr.addr("MCD_IAM_AUTO_LINE"));
 
+    // Specific for this spell
+
+    address constant GRO_MULTISIG        = 0x7800C137A645c07132886539217ce192b9F0528e;
```

Comment:
Are these subject to change? Otherwise it might make sense to store them in `addresses_mainnet.sol`